submissions/m-ruslan/memory-pair-game/app.js

```diff
@@ -0,0 +1,135 @@
+// main logic of game
+document.addEventListener("DOMContentLoaded", () => {
+  let cardID, cardValue, prevCardId, prevCardValue;
+  let qOfOpenCards = 0;
+
+  startNewGame();
+
+  document.querySelector("main").addEventListener("click", (event) => {
+    let aEl = event.target.closest(".flip-container");
+    if (!aEl) return;
+    if (aEl.id === prevCardId && aEl.id !== undefined) return;
```

Comment:
combine this if statements
and read about [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) you can try to use it in this case